src/ray/gcs/redis_context.h

```diff
@@ -73,7 +73,7 @@ class RedisContext {
                   int log_length = -1);
 
   Status SubscribeAsync(const ClientID &client_id, const TablePubsub pubsub_channel,
-                        const RedisCallback &redisCallback);
+                        const RedisCallback &redisCallback, int64_t *out_callback_index);
   redisAsyncContext *async_context() { return async_context_; }
```

Comment:
int64_t &out_callback_index

Reply:
Previously in [other place](https://github.com/apache/arrow/pull/2174#discussion_r199961035) I used reference instead of pointer, but Philipp pointed out that it conflicts with [Google's style](https://google.github.io/styleguide/cppguide.html#Reference_Arguments)... However, point may be a better choice, when the caller doesn't want to create a useless int64 to call this function if the caller doesn't want to use this index.